docs/speech-usage.rst

```diff
@@ -83,5 +83,73 @@ data to text and returns alternative text transcriptons.
      transcript: Hello, this is one test
      confidence: 0
 
+
+Streaming Recognition
+---------------------
+
+The :meth:`~google.cloud.speech.Client.stream_recognize` method converts speech
+data to possible text alternatives on the fly.
+
+.. note::
+    Streaming recognition requests are limited to 1 minute of audio.
+
+    See: https://cloud.google.com/speech/limits#content
+
+.. code-block:: python
+
+    >>> from google.cloud import speech
+    >>> client = speech.Client()
+    >>> results = client.stream_recognize('hello.flac', 'FLAC', 44100)
+    >>> results[0].is_final
+    False
+    >>> results[2].is_final
+    True
+    >>> results[2].alternatives[0].transcript
+    hello
+    >>> results[2].alternatives[0].confidence
+    0.96976006031
+
+For continuous speech containing more than one word, the ``single_utterance``
+option should be disabled.
```

Comment:
I don't mind the logic. Just make it clear in these docs what the default is. In both examples you've used an explicit value. Instead, use no value (i.e. the default) in one of the example and then be explicit with a non-default in the other example.